src/unix/bsd/netbsdlike/netbsd/riscv64.rs

```diff
@@ -26,7 +26,7 @@ s_no_extra_traits! {
     }
 }
 
-pub(crate) const _ALIGNBYTES: usize = ::mem::size_of::<c_long>() - 1;
+pub(crate) const _ALIGNBYTES: usize = core::mem::size_of::<c_long>() - 1;
```

Comment:
Actually maybe just hold off until I can merge https://github.com/rust-lang/libc/pull/4161, that will make this a bit easier since it can just be `mem::size_of`.

Reply:
I stand corrected, this actually shouldn't be a problem.